roda-ui/roda-wui/src/main/java/org/roda/wui/api/controllers/UserLogin.java

```diff
@@ -12,6 +12,7 @@
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServletRequest;
 
+import com.google.gwt.core.client.GWT;
```

Comment:
this import appears to be unnecessary, please check and run the code reformat command on all files that have been changed